types/circular-dependency-plugin/index.d.ts

```diff
@@ -8,22 +8,44 @@ import { Plugin, Module, compilation } from 'webpack';
 
 export = CircularDependencyPlugin;
 
+/**
+ * Detect modules with circular dependencies when bundling with webpack.
+ */
 declare class CircularDependencyPlugin extends Plugin {
   constructor(options?: CircularDependencyPlugin.Options);
   // Not exposing `isCyclic` because it isn't meant to be public, I believe
 }
 
 declare namespace CircularDependencyPlugin {
   interface Options {
+    /**
+     * @default false
+     */
     allowAsyncCycles?: boolean;
+    /**
+     * @default process.cwd()
+     */
     cwd?: string;
+    /**
+     * @default /$^/
+     */
     exclude?: RegExp;
+    /**
+     * @default /.*\/
+     */
+    include?: RegExp;
+    /**
+     * @default false
+     */
     failOnError?: boolean;
-    onDetected?: (x: {
+    /**
+     * @default false
+     */
+    onDetected?: false | ((x: {
```

Comment:
Odd as it looks, this does seem to match the underlying library.  Thanks for the link!